### PR TITLE
Update Helpdesk for Giza

### PR DESCRIPTION
**Pull request by bwhm**
Note that for the early birds. you need to switch to the `giza` branch on the [joystream](https://github.com/Joystream/joystream) repo, as we haven't merged to master.

### CODE REVIEW

File: tools/query-node/README.md [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+
+### Set Your Environment
+```
+# If your local node is synced:
+$ export WS_PROVIDER_ENDPOINT_URI=ws://localhost:9944
+# If not:
+$ export WS_PROVIDER_ENDPOINT_URI=wss://rome-rpc-endpoint.joystream.org:9944
+
+$ export QUERY_NODE_URL=https://hydra-sumer.joystream.org/graphql
+$ export AT_BLOCK_NUMBER=<xxx>
```
Comment (mnaamani) [inline tools/query-node/README.md:56]: we should probably clarify that this should be the runtime update block number,

File: tools/query-node/README.md [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+$ yarn workspace query-node-mappings bootstrap-data:fetch
+$ yarn workspace query-node-root build
+```
+
+### Update Manifest
+Now, you need to update the `manifest.yml`:
+
+```
+cd query-node/
+yarn build
```
Comment (mnaamani) [inline tools/query-node/README.md:70]: I don't think this additional build step is necessary

File: tools/query-node/README.md [hunk 3]
```diff
@@ -0,0 +1,271 @@
...
+nano manifest.yml
+
+# change second to last line from:
+#           height: "[0,0]"
+# to:
+           height: "[<xxx>,<xxx>]"
```
Comment (mnaamani) [inline tools/query-node/README.md:76]: same about xxxx

**Review by mnaamani — CHANGES_REQUESTED** [review]
Awesome job putting this guide together.
Only a couple of typos and had a suggestion for query-node setup. Basically installing latest docker-compose will make instructions much simpler.

One final thought - I think it doesn't make sense for us to require to keep running old sumer query-node just for purpose of getting the initial state of channel and video categories. So we should probably just copy those somewhere and use them in the instructions.

File: tools/query-node/README.md [hunk 4]
```diff
@@ -0,0 +1,271 @@
...
+```
+
+### Prepare Deployement
+We assume that:
+- you may want to run a `storage-node` or `distributor-node` in the future; and
+- `docker-compose --version` returns a version lower than `1.29.0`.
+  - If it is `1.29.0` or higher, you don't have to make these changes.
+
```
[the remark this answers is not in the document]
Comment (bwhm) [reply tools/query-node/README.md:84]: Thanks - will add :)

File: tools/query-node/README.md (hunk 1 above)
Comment (bwhm) [reply tools/query-node/README.md:56]: Yeah, I had the `<xxx>` to search/replace, but...

File: tools/query-node/README.md (hunk 2 above)
Comment (bwhm) [reply tools/query-node/README.md:70]: Thanks!

File: tools/query-node/README.md (hunk 1 above)
Comment (bwhm) [reply tools/query-node/README.md:56]: ...forgot to push!

**Comment by bwhm** [comment]
@mnaamani 
I think this needs more additions and improvements, but it's a start, so if ok -> merge?

File: roles/distributors-lead/README.md [hunk 5]
```diff
@@ -0,0 +1,600 @@
...
+A bag can be added to, and removed from, a bucket at any time, as long as it doesn't violate some other parameter.
+
+Note that this means a bag can be removed from all buckets, which makes it hard (if not impossible) to recover that data. Therefore, the Lead should never remove a bag from a bucket without knowing it is held (not "just" supposed to be held) by other buckets.
+
+
+### Distribution Buckets
+The distribution buckets work somewhat differently, as they also have the concept of `bucket families`.
+
+#### Bucket Families
+A bucket family is a collection of distribution buckets associated with a geographical region (eg. from larger to smaller: World, Europe, Norther Europe, Scandinavia, Norway, Oslo). This is meant to reduce latency when serving content.
```
Comment (blrhc) [inline roles/distributors-lead/README.md:85]:
```suggestion
A bucket family is a collection of distribution buckets associated with a geographical region (eg. from larger to smaller: World, Europe, Northern Europe, Scandinavia, Norway, Oslo). This is meant to reduce latency when serving content.
```

File: roles/distributors-lead/README.md [hunk 6]
```diff
@@ -0,0 +1,600 @@
...
+    - [update-bucket-mode](#update-bucket-mode)
+  - [update-bucket-status](#update-bucket-status)
+    - [update-dynamic-bag-policy](#update-dynamic-bag-policy)
+  - [Operator](#operator)
+    - [accept-invitation](#accept-invitation)
+    - [set-metadata](#set-metadata)
+<!-- TOC END -->
+
+## The Storage and Distribution System
+Storage and Distribution system is well documented on github. A simple version will be outlined here.
```
Comment (blrhc) [inline roles/distributors-lead/README.md:37]:
```suggestion
Storage and Distribution system is well documented on GitHub. A simple version will be outlined here.
```

File: roles/distributors/README.md [hunk 7]
```diff
@@ -0,0 +1,375 @@
...
+Now, get the additional dependencies:
+```
+$ apt-get update && apt-get upgrade -y
+```
+
+## Setup Hosting
+In order to allow for users to upload and download, you have to setup hosting, with an actual domain as both Chrome and Firefox requires `https://`. If you have a "spare" domain or subdomain you don't mind using for this purpose, go to your domain registrar and point your domain to the IP you want. If you don't, you will need to purchase one.
+
+### Caddy
+To configure SSL-certificates the easiest is to use [caddy](https://caddyserver.com/), but feel free to take a different approach. Note that if you are using caddy for commercial use, you need to acquire a license. Please check their terms and make sure you comply with what is considered personal use.
```
Comment (blrhc) [inline roles/distributors/README.md:72]:
```suggestion
To configure SSL-certificates the easiest option is to use [caddy](https://caddyserver.com/), but feel free to take a different approach. Note that if you are using caddy for commercial use, you need to acquire a license. Please check their terms and make sure you comply with what is considered personal use.
```

File: tools/query-node/README.md [hunk 8]
```diff
@@ -0,0 +1,271 @@
...
+```
+
+Finally, if you included hosting of the `Query-node`, you can access the graphql server at `https://<your.cool.url>/server/graphql`.
+Note that you'd need to change `https://<your.cool.url>/graphql` address to `https://<your.cool.url>/server/graphql` as well for the server to be reached.
+
+## Setup Hosting
+In order to allow for users to upload and download, you have to setup hosting, with an actual domain as both Chrome and Firefox requires `https://`. If you have a "spare" domain or subdomain you don't mind using for this purpose, go to your domain registrar and point your domain to the IP you want. If you don't, you will need to purchase one.
+
+### Caddy
+To configure SSL-certificates the easiest is to use [caddy](https://caddyserver.com/), but feel free to take a different approach. Note that if you are using caddy for commercial use, you need to acquire a license. Please check their terms and make sure you comply with what is considered personal use.
```
Comment (blrhc) [inline tools/query-node/README.md:201]:
```suggestion
To configure SSL-certificates the easiest option is to use [caddy](https://caddyserver.com/), but feel free to take a different approach. Note that if you are using caddy for commercial use, you need to acquire a license. Please check their terms and make sure you comply with what is considered personal use.
```

**Review by blrhc — APPROVED** [review]
Just very minor typos/style, other than that, very clear AFAIC.

**Pull request merged**